lib/remote-template.js

```diff
@@ -338,7 +340,10 @@ async function makeGitHubApiRequest(options, url, handleNotFound) {
     // TODO(@lishaduck) [engine:node@>=21]: We can use `fetch` now.
     const response = await got(url, parameters);
 
-    return response.body;
+    /** @type {{body: unknown}} */
```

Comment:
Seeing this over and over, I feel like `TS` should try to understand type annotations on a return statement like
```ts
/** @type {{body: unknown}} */
return response.body;
```

Reply:
It's actually a ts-eslint issue: typescript-eslint/typescript-eslint#1682. It can't read JSDoc casts b/c they're technically just comments, so the type queries are wrong. `no-unsafe-return` bans returning `any`, which is good, but can't read immediate casts, it needs an intermediate variable. In normal TS, you can just use `as`.